src/lib/.eslintrc

```diff
@@ -1,5 +1,6 @@
 {
```

Comment:
Is it necessary to have several eslintrc files across the project??

Reply:
yes, actually some parts are less strict than others, for instance, `console.log` is allowed in some parts, or, actually I do not care about return type in unit test functions, for the other side, there was so many errors perhaps some `.eslintrc` file are just for display warnings and future clean up phase.

Reply:
Those files basically override or extends the root configuration.